ring/lifecycler.go

```diff
@@ -913,6 +923,7 @@ func (i *Lifecycler) updateCounters(ringDesc *Desc) {
 	healthyInstancesCount := 0
 	instancesCount := 0
 	zones := map[string]int{}
+	healthyZones := map[string]int{}
```

Comment:
This is confusing, as it's counting healthy instances per zone, not healthy zones.